config/webpack.dev.config.js

```diff
@@ -124,26 +125,25 @@ module.exports = merge(commonConfig, {
       },
       {
         test: /\.(jpe?g|png|gif)(\?v=\d+\.\d+\.\d+)?$/,
-        use: [
-          'file-loader',
-          {
-            loader: 'image-webpack-loader',
-            options: {
-              mozjpeg: {
-                progressive: true,
-                quality: 65,
-              },
-              gifsicle: {
-                interlaced: false,
-              },
-              pngquant: {
-                quality: [0.65, 0.90],
-                speed: 4,
-              },
+        loader: 'file-loader',
+      },
+    ],
+  },
+  optimization: {
+    minimizer: [
+      '...',
+      new ImageMinimizerPlugin({
+        minimizer: {
+          implementation: ImageMinimizerPlugin.sharpMinify,
+          options: {
+            encodeOptions: {
+              ...['png', 'jpeg', 'jpg'].reduce((accumulator, value) => (
+                { ...accumulator, [value]: { progressive: true, quality: 65 } }
```

Comment:
[clarification/curious] Is the `progressive: true` applicable to all file extensions? In the previous implementation, `progressive` is only relevant to `mozjpeg`.

Reply:
`jpeg` & `jpg` seems to direct to same options, `png` also has option for progressive (interlace) scan 
<img width="787" alt="Screen Shot 2022-09-09 at 2 31 40 PM" src="https://user-images.githubusercontent.com/59555732/189319974-2b02e4a3-3896-43ec-9015-de191fc0bea4.png">